dist/rpm/input-leap.spec.in

```diff
@@ -32,6 +33,10 @@ BuildRequires: pkgconfig(Qt6Linguist)
 BuildRequires: make
 BuildRequires: pkgconfig(libei-1.0)
 BuildRequires: pkgconfig(libportal) >= 0.8
+%if %{gmock_gtest_installed}
+BuildRequires: pkgconfig(gmock)
+BuildRequires: pkgconfig(gtest)
+%endif
```

Comment:
BuildRequires are used to decide which packages to install. Here you’re forcing install only if already installed, assuming you can run `FetchContent` which is not always the case. Just always include the `BuildRequires:`

Reply:
did that